src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/services/featureManagerWrapper.js

```diff
@@ -0,0 +1,33 @@
+var moduleName = "virtoCommerce.DemoSolutionFeaturesModule";
+
+angular.module(moduleName)
+    .factory('virtoCommerce.demoFeatures.featureManagerSubscriber', ['$rootScope', 'virtoCommerce.demoFeatures.featureManager', function ($rootScope, featureManager) {
+        var result = {};
+
+        result.callbackCollection = [];
```

Comment:
Just "callbacks" may be?

Reply:
Renamed to 'callbacksGroupedByFeatureName' since we also have feature names there